### PR TITLE
update zenodo file

### PR DESCRIPTION
**Pull request by psychocoderHPC**
- add contributor section
- fix indentions

### CODE REVIEW

**Review by bernhardmgruber — APPROVED** [review]

**Comment by psychocoderHPC** [comment]
@theZiz, @MarkusVelten, @bertwesarg, @tdd11235813 please check your entry, e.g. affiliation during the contribution.

File: .zenodo.json [hunk 1]
```diff
@@ -1,30 +1,48 @@
...
+         "orcid":"0000-0003-1642-0459"
+      }
+   ],
+   "contributors":[
+      {
+         "name":"Wesarg, Bert",
+         "affiliation":"TU Dresden"
+      },
+      {
+         "name":"Velten, Markus"
```
Comment (MarkusVelten) [inline .zenodo.json]: Thank you for the information, please add my affiliation:

```suggestion
         "name":"Velten, Markus",
         "affiliation":"TU Dresden"
```

**Review by MarkusVelten — COMMENTED** [review]
Please add my affiliation as suggested.

File: .zenodo.json [hunk 2]
```diff
@@ -1,30 +1,48 @@
...
+   "description":"LLAMA is a C++17 template header-only library for the abstraction of memory access patterns. It distinguishes between the view of the algorithm on the memory and the real layout in the background. This enables performance portability for multicore, manycore and gpu applications with the very same code.",
+   "creators":[
+      {
+         "affiliation":"Helmholtz-Zentrum Dresden-Rossendorf, TU Dresden",
+         "name":"Matthes, Alexander",
+         "orcid":"0000-0002-6702-2015"
+      },
+      {
+         "affiliation":"Helmholtz-Zentrum Dresden-Rossendorf, CASUS, CERN",
+         "name":"Gruber, Bernhard Manfred"
```
Comment (bernhardmgruber) [inline .zenodo.json]: Please add my orcid as well! Thx!

```suggestion
         "name":"Gruber, Bernhard Manfred",
         "orcid":"0000-0001-7848-1690"
```

**Comment by psychocoderHPC** [comment]
updated

File: .zenodo.json [hunk 3]
```diff
@@ -1,30 +1,50 @@
...
+   "creators":[
+      {
+         "affiliation":"Helmholtz-Zentrum Dresden-Rossendorf, TU Dresden",
+         "name":"Matthes, Alexander",
+         "orcid":"0000-0002-6702-2015"
+      },
+      {
+         "affiliation":"Helmholtz-Zentrum Dresden-Rossendorf, CASUS, CERN",
+         "name":"Gruber, Bernhard Manfred",
+	 "orcid":"0000-0001-7848-1690"
```
Comment (bernhardmgruber) [inline .zenodo.json]: Please fix the whitespace here

File: .zenodo.json [hunk 4]
```diff
@@ -1,30 +1,50 @@
...
+      }
+   ],
+   "contributors":[
+      {
+         "name":"Wesarg, Bert",
+         "affiliation":"TU Dresden"
+      },
+      {
+         "name":"Velten, Markus",
+	 "affiliation":"TU Dresden"
```
Comment (bernhardmgruber) [inline .zenodo.json]: Please fix the whitespace here

**Review by bernhardmgruber — CHANGES_REQUESTED** [review]

**Review by bernhardmgruber — APPROVED** [review]

**Pull request merged**